.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,31 @@
 ---
-name: I want to report an issue with React Native.
+name: 🐛 Bug Report
 about: You want to report a reproducible bug or regression in React Native.
-labels: "Bug Report"
+labels: "Type: Bug Report"
 ---
 
-<!--
-  If you are looking for help, please review the documentation or ask a question on Stack Overflow:
-  - https://facebook.github.io/react-native
-  - https://stackoverflow.com/questions/tagged/react-native
+## 🐛 Bug Report
 
-  If you want to report a problem with React Native itself, please fill out the template.
+A clear and concise description of what the bug is.
```

Comment:
Please bring back the comments for all the non-essential stuff in the template that only the creator of the issue should see but not the people who look at the issues.